TypeScriptException on required decimal property

Deployment of the following script with AdminGui fails with TypeScriptException.

```
Module TestAdminGuid
{
    Entity TestE
    {
        Decimal SomeValue { Required; }
    }
}
```

Error log:

```
[Trace] DeployPackages: > adminGui@1.0.0 tsc C:\My Projects\Rhetos\Source\Rhetos\Resources\AdminGuiCompile
[Trace] DeployPackages: > tsc
[Trace] DeployPackages:
[Trace] DeployPackages: scripts/models/rhetos.angular2.ts(452,17): error TS1005: ',' expected.
```

Rhetos.Angular2.ts:

```typescript
448    getValidators(): { [propName: string]: ValidatorDefinition[]; } {
449        return {
450            'SomeValue':[
451                {Validator: RegexValidatorFactory('^-?[0-9]+(\\.[0-9]+)?$'), ErrorCode: 'notMatchingRegex', ErrorMessage: 'Example: 100.0, 100.10 '}
452                {Validator :Validators.required, ErrorCode: 'required', ErrorMessage: 'SomeValue is required' }
453            ]
454        };
455    }
```

--- a/Plugins/AdminGuiPlugin/Angular2ModelGenerator/Templates/ValidatorTemplates.cs
+++ b/Plugins/AdminGuiPlugin/Angular2ModelGenerator/Templates/ValidatorTemplates.cs
@@ -28,7 +28,7 @@
 
         public static string Decimal(string regex, string errorMessage)
         {
-            return $@"{{Validator: RegexValidatorFactory('{regex}'), ErrorCode: 'notMatchingRegex', ErrorMessage: '{errorMessage}'}}
+            return $@"{{Validator: RegexValidatorFactory('{regex}'), ErrorCode: 'notMatchingRegex', ErrorMessage: '{errorMessage}'}},
                 ";
         }
     }

